SECP-service/src/main/resources/assets/app/directives/chat/chat-message.html

```diff
@@ -0,0 +1,30 @@
+<div>
```

Comment:
The use of directives is a lot better. Thanks. The main chat page was getting a little out of hand.

Reply:
True. my next task is to make the contact list on the chat page as a css too. :)